softlearning/environments/utils.py

```diff
@@ -8,15 +8,22 @@
     DmControlAdapter,
 )
 
+from .adapters.robosuite_adapter import (
```

Comment:
One thing to do here is that we need to make all these adapters fail lazily if they can't be imported, such that not all the different environments need to be installed.